homeassistant/components/mqtt/__init__.py

```diff
@@ -9,6 +9,8 @@
 import logging
 import os
 import socket
+import json
+import jsonpath_rw
```

Comment:
Home Assistant installs dependencies on demand. Can you move this import inside the constructor of the `_JsonFmtParser` class and add it to `REQUIREMENTS` ?